app/code/Transoft/Blog/Block/Adminhtml/Post/Edit/BackButton.php

```diff
@@ -0,0 +1,39 @@
+<?php
+declare(strict_types=1);
+namespace Transoft\Blog\Block\Adminhtml\Post\Edit;
+
+use Magento\Cms\Block\Adminhtml\Block\Edit\GenericButton;
+use Magento\Framework\View\Element\UiComponent\Control\ButtonProviderInterface;
+
+/**
+ * Class BackButton
+ *
+ * Generates back button
+ */
+class BackButton extends GenericButton implements ButtonProviderInterface
+{
+    /**
+     * Get button params
+     *
+     * @return array
+     */
```

Comment:
Please use @inheritDoc to define a description for interface implemented methods or overridden methods. Fix in **all** files

Reply:
fixed